### PR TITLE
Standardize the ug

### PR DESCRIPTION
**Pull request by nickeltea**
The ug does not look standardized.

The ug should look like it was written by one person.

Let's make the format of the ug standardized.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -301,9 +303,12 @@ Format: `addT d/DESCRIPTION D/DEADLINE`
 
 * Tasks that are past their deadline can still be added since overdue tasks might have to be completed still.
 
-<div markdown="span" class="alert alert-warning">:warning: Adding a task that is the same as one already in the address book. Two tasks are the same if they have the same description, deadline and labels.
+<div markdown="block" class="alert alert-warning">:warning: There will be an error if you:<br>
+
```
Comment (feliciagan) [inline docs/UserGuide.md:307]: Add "Do not adhere to field constraints(link to section 2 intro)"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -461,15 +474,15 @@ Format: `unarchiveT INDEX`
...
 
 Format: `remindT DEADLINE`
 
 * Both completed and incomplete tasks are listed.
 
 * Task with deadlines that are already past are also listed.
 
-<div markdown="span" class="alert alert-info">:bulb:Tasks that are complete are also listed so users are reminded to delete them.
+<div markdown="span" class="alert alert-info">:bulb:Tasks that are complete are also listed so users are reminded to delete or archive them.
 </div>
```
Comment (feliciagan) [inline docs/UserGuide.md:489]: Add incorrect usage box with bulletpoint
"Do not adhere to deadline constraints. (link to section 2 intro)

Comment (feliciagan) [reply docs/UserGuide.md:489]: wait sorry dont link to section 2 intro, instead just write it must be dd-mm-yyyy format since its just one field

**Comment by alvintfl** [comment]
LGTM

**Pull request merged**